databases/postgresql-activerecord.md

```diff
@@ -0,0 +1,139 @@
+# PostgreSQL
+
+[PostgreSQL](http://www.postgresql.org/) is an object-relational database management system (ORDBMS).
+
+The connection between PostgreSQL and Ruby happens via the [`pg`](https://rubygems.org/gems/pg) gem.
+
+## Installation
+
+[Install PostgreSQL](http://www.postgresql.org/download/)
+
+
+Install Ruby Postgres bridge
+```
+gem install pg
+```
+
+*Note:* OSX Users may have troubles with the installation of the `pg` gem. Read [this](http://stackoverflow.com/a/19850273/2980299)
+
+
+```
+gem install activerecord
+gem install sinatra-activerecord
+gem install rake # to apply migrations
+```
+
+
+## Connect to the database
+
+Either by making a `config/database.yml` file that gets automatically loaded
+
+```
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: database_name
+  pool: 2
+  username: your_username
+  password: your_password
+
+production:
+  adapter: postgresql
+  encoding: unicode
+  database: database_name
+  pool: 2
+  username: your_username
+  password: your_password
+```
+
+or by specifying in your sinatra app
+
+```ruby
+configure :development do
+  set :database, {adapter: 'postgres',  encoding: 'unicode', database: 'your_database_name', pool: 2, username: 'your_username', password: 'your_password'}
+end
+
+configure :production do
+  set :database, {adapter: 'postgres',  encoding: 'unicode', database: 'your_database_name', pool: 2, username: 'your_username', password: 'your_password'}
+end
```

Comment:
Are you sure this is going to work? Doesn't AR's migration `rake` helper require the `config/database.yml` file to be present? If AR does recognize Sinatra's configuration helper, then there probably should be a specific way to load it into the `Rakefile`.